AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -3105,7 +3105,6 @@ animenewsnetwork.com#@##Adbanner
 @@||yandex.ru/ads/system/context.js$xmlhttprequest,domain=sdamgia.ru
 timeanddate.com#@##ad-wrap2
 lite.liiingo.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
-biolink.webtena.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
```

Comment:
Both rules are ok) Just add to https://github.com/AdguardTeam/AdguardFilters/blob/5136e7865d95a5882c81da8aa35795bea7e236f8/BaseFilter/sections/antiadblock.txt#L1075

```
! https://github.com/AdguardTeam/AdguardFilters/issues/176432
biolink.webtena.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
!+ PLATFORM(ios, ext_android_cb)
@@||biolink.webtena.com^$generichide
```

Reply:
lite.liiingo.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
is equally misplaced